tensorflow/inference/docker/2.4/py3/cu110/Dockerfile.gpu

```diff
@@ -0,0 +1,174 @@
+FROM nvidia/cuda:11.0-base-ubuntu18.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG PYTHON=python3.7
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.7.7
+ARG OPENSSL_VERSION=1.1.1g
+
+ARG TFS_SHORT_VERSION=2.4
+ARG TFS_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/serving/r2.4_aws/20210125-223221/gpu/py37/tensorflow_model_server
+
+ENV NCCL_VERSION=2.7.6-1+cuda11.0
+ENV CUDNN_VERSION=8.0.4.30-1+cuda11.0
+ENV TF_TENSORRT_VERSION=7.1.3
+
+# See http://bugs.python.org/issue19846
+ENV LANG=C.UTF-8
+ENV PYTHONDONTWRITEBYTECODE=1
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+ENV PYTHONUNBUFFERED=1
+ENV SAGEMAKER_TFS_VERSION="${TFS_SHORT_VERSION}"
+ENV PATH="$PATH:/sagemaker"
+ENV MODEL_BASE_PATH=/models
+# The only required piece is the model name in order to differentiate endpoints
+ENV MODEL_NAME=model
+# Fix for the interactive mode during an install in step 21
+ENV DEBIAN_FRONTEND=noninteractive
+
+# allow unauthenticated and allow downgrades for special libcublas library
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends --allow-unauthenticated --allow-downgrades\
+    ca-certificates \
+    cuda-command-line-tools-11-0 \
+    cuda-nvrtc-11-0 \
+    cuda-nvrtc-dev-11-0 \
+    cuda-cudart-dev-11-0 \
+    libcufft-dev-11-0 \
+    libcurand-dev-11-0 \
+    libcusolver-dev-11-0 \
+    libcusparse-dev-11-0 \
+    libbz2-dev \
+    liblzma-dev \
+    #cuda-cublas-dev not available with 10-1, install libcublas instead
+    libcublas-11-0=11.2.0.252-1 \
+    libcublas-dev-11-0=11.2.0.252-1 \
+    libcudnn8=${CUDNN_VERSION} \
+    libcudnn8-dev=${CUDNN_VERSION} \
+    libnccl2=${NCCL_VERSION} \
+    libnccl-dev=${NCCL_VERSION}  \
+    libgomp1 \
+    libffi-dev \
+    curl \
+    emacs \
+    git \
+    wget \
+    vim \
+    build-essential \
+    zlib1g-dev \
+    openssl \
```

Comment:
same here. do we need `openssl` and `libssl1.1` ?

Reply:
Install openssl and libssl to resolve https://ubuntu.com/security/CVE-2020-1971